FV3/atmos_cubed_sphere/makefile

```diff
@@ -23,7 +23,7 @@ endif
 
 LIBRARY  = libfv3core.a
 
-FFLAGS   += -I$(FMS_DIR) -I../gfsphysics -I../ipd -I../io -I../namphysics
+FFLAGS   += -I$(FMS_DIR) -I../coarse_graining -I../gfsphysics -I../ipd -I../io -I../namphysics
```

Comment:
It's true; I guess I've been thinking more about our own needs at the moment.  If we were to contribute things upstream, my sense is that eventually what goes in the coarse_graining module would live in FMS, in which case we wouldn't need to change this makefile at all.  For now though, based on discussion with @mcgibbon, I chose to keep things separate, so I needed to include it here.

I think it would be straightforward if/when the time comes to make that change.  Do you think that's ok, or should we strive to make that decision now?

Reply:
I think that's ok! It can always be merged down the road.